assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -151,6 +170,17 @@ const OrderSummaryItem = ( { cartItem } ) => {
 					variation={ variation }
 				/>
 			</div>
+			<span className="screen-reader-text" tabIndex={ 0 }>
+				{ sprintf(
+					/* translators: Total Price for %d quantity items: %s */
+					__(
+						'Total Price for %1$d items: %2$s',
```

Comment:
Ideally, we should use sentence case everywhere, even if it's a text for a screen reader:
```suggestion
						'Total price for %1$d items: %2$s',
```

Reply:
Another thing about this string, you will need to account for the case when there is only one item of a specific product. In that case `items` should be `item`. For this, we use `sprintf()` and `_n()`.

You can see an example of how this functions are being used here:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/19d29d11fea059650da48cbca4f35275e88324d6/assets/js/editor-components/product-category-control/index.js#L48-L58

Reply:
That's great, suggestion has been applied here as well. Thanks!

Reply:
One mention is that I've added the product name in the mix as it makes the Screen Reader easier to understand on my part, so the user gets the full info of quantity, product name, and total price in one sentence.